### PR TITLE
Change validate to diagnose

### PR DESCRIPTION
**Pull request by tpantelis**

### CODE REVIEW

**Review by mangelajo — APPROVED** [review]

File: pkg/subctl/cmd/validate_fw_vxlan.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ import (
 
 var validateFirewallVxLANConfigCmd = &cobra.Command{
 	Use:   "vxlan",
-	Short: "Validate if firewall allows VxLAN traffic within the cluster.",
-	Long:  "This command checks whether firewall configuration allows traffic via the Submariner VxLAN interface.",
+	Short: "Check firewall access for Submariner VxLAN traffic",
```
Comment (nyechiel) [inline pkg/subctl/cmd/validate_fw_vxlan.go]:
```suggestion
	Short: "Check firewall access for Submariner VXLAN traffic",
```

File: pkg/subctl/cmd/validate_fw_vxlan.go [hunk 2]
```diff
@@ -29,8 +29,8 @@ import (
...
+	Long:  "This command checks if the firewall configuration allows traffic via the Submariner VxLAN interface.",
```
Comment (nyechiel) [inline pkg/subctl/cmd/validate_fw_vxlan.go]:
```suggestion
	Long:  "This command checks if the firewall configuration allows traffic via the Submariner VXLAN interface.",
```

**Review by nyechiel — COMMENTED** [review]
One nit, but otherwise LGTM

**Review by nyechiel — APPROVED** [review]

File: pkg/subctl/cmd/validate.go [hunk 3]
```diff
@@ -21,9 +21,9 @@ import (
 
 var (
 	validateCmd = &cobra.Command{
```
Comment (Jaanki) [inline pkg/subctl/cmd/validate.go:23]: Don't we have to change this also? and package name too? Or am I missing something?

**Review by Jaanki — COMMENTED** [review]
Are we not renaming package as well?

File: pkg/subctl/cmd/validate.go (hunk 3 above)
Comment (tpantelis) [reply pkg/subctl/cmd/validate.go:23]: yeah we talked about this on slack - all of that is internal - I'll push a PR to rename the files, vars etc later - let's get the external changes in for 0.9.0.

**Review by dfarrell07 — APPROVED** [review]

**Comment by tpantelis** [comment]
@sridhargaddam I'd like to get your review as well.

**Review by mkolesnik — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**